tests/Console/Scheduling/FrequencyTest.php

```diff
@@ -0,0 +1,167 @@
+<?php
+
+use Illuminate\Console\Scheduling\ManagesFrequencies;
+use Mockery as m;
+use PHPUnit\Framework\TestCase;
+
+class FrequencyTest extends TestCase
+{
+    /**
+     * @var \Illuminate\Console\Scheduling\Event
+     */
+    protected $event;
+
+    public function setUp()
+    {
+        /**
+         * Fake the Illuminate\Console\Scheduling\Event class
+         */
+        $this->event = new class {
+            use ManagesFrequencies;
+
+            public $expression = '* * * * * *';
+
+            public function getExpression()
+            {
+                return $this->expression;
+            }
+        };
+    }
+
+    /**
+     * @test
+     */
+    function every_minute()
```

Comment:
Yeah, its my macro for creating tests. I use it for my personal tests, and out of habit used them here. My mistake, should have noticed that :)